core/domain/stats_domain_test.py

```diff
@@ -812,14 +812,19 @@ def test_cannot_update_exp_issue_with_no_schema_version(self):
         exp_issues_model = stats_models.ExplorationIssuesModel.get_model(
             'exp_id', 1)
 
-        with self.assertRaises(Exception):
+        with self.assertRaisesRegexp(
+            Exception,
+            r'unsupported operand type\(s\) for \+=: \'NoneType\' and \'int\''):
```

Comment:
This looks like an unexpected python error, can you please check whether this is in an expected error?